src/core/utils/assets.ts

```diff
@@ -196,7 +197,7 @@ export function parseUserAsset({
   balance,
   smallBalance,
 }: {
-  asset: ZerionAsset | AssetApiResponse;
+  asset: ZerionAsset | AssetApiResponse | AddysPositionAsset;
```

Comment:
is it possible to remove ZerionAsset ? i started removing it from https://github.com/rainbow-me/browser-extension/pull/1421